src/engine/client/cg_api.h

```diff
@@ -86,9 +86,30 @@ enum class MouseMode
 	SystemCursor, // The input is sent as positions, the cursor should be rendered by the system
 };
 
+#define MAX_MARK_FRAGMENTS 128
+#define MAX_MARK_POINTS 384
+
+struct fragmentPacket_t
```

Comment:
The structs are related to the "mark" feature and so ought to have "mark" in their names. This sounds like something about the network.

Reply:
I renamed `fragmentPacket_t` and `fragmentBuffer_t` to `markMsgInput_t` and `markMsgOutput_t`.